### PR TITLE
fix(styles): update tile title font size [ci visual]

### PR DESCRIPTION
**Pull request by droshev**
## Related Issue
Closes SAP/fundamental-styles#4502

update tile title font size

### CODE REVIEW

**Review by g-cheishvili — APPROVED** [review]

File: packages/styles/src/tile.scss [hunk 1]
```diff
@@ -209,7 +209,8 @@ $fd-line-tile-badge-horizontal-spacing: 0.375rem !default;
     @include fd-line-clamp();
 
     color: var(--sapTile_TitleTextColor);
-    font-size: var(--sapFontHeader5Size);
+    font-size: var(--sapFontHeader6Size);
```
Comment (InnaAtanasova) [inline packages/styles/src/tile.scss:212]: that's the font-size for small Tile

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: packages/styles/src/tile.scss (hunk 1 above)
Comment (droshev) [reply packages/styles/src/tile.scss]: updated

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
The values are the same for all themes in Horizon and all themes in Quartz. You need to put them in the `common` folder and have only 2 files, 1 for Horizon and 1 for Quartz, instead of 9 files

**Comment by droshev** [comment]
> The values are the same for all themes in Horizon and all themes in Quartz. You need to put them in the `common` folder and have only 2 files, 1 for Horizon and 1 for Quartz, instead of 9 files

done

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**